package.json

```diff
@@ -12,12 +12,13 @@
     "postbuild:client": "mkdir -p dist/client/public && cp _redirects dist/client/_redirects && cp public/favicon.ico dist/client/favicon.ico && cp public/android-chrome-512x512.png dist/client/public/icon.png && cp -r .well-known dist/client/.well-known",
     "build:server": "cp -r src/lambda/ dist/server",
     "postbuild:server": "mkdir -p dist/server/server && mv dist/client/index.html dist/server/server/",
-    "lint": "eslint 'src/**/*.js' --quiet --fix",
+    "lint": "eslint . --quiet --fix",
```

Comment:
`eslint` can only parse `.js(x)`/`.ts(x)`-files, right? 🤔

Reply:
yes

Reply:
So we shouldn't run `eslint` on all files (`.`) then, right? 🤔

Reply:
Now there are only js files. The default extension for eslint is JavaScript file